changelog.txt

```diff
@@ -22,6 +22,7 @@ new features
 - detector description is saved to nur output files
 - new handling of random numbers. Each module has its own random generator instance. Global seed can be controlled via 
   config file setting. 
+- More accurate timing for the ARZ models
```

Comment:
the timing is not more accurate, you just redefine the trace start time. I'd rather name the change "trace start time adjusted such that global time of pulse position corresponds to propagation time"

Reply:
Sure, you're right. Changed.